app/org/sagebionetworks/bridge/dynamodb/DynamoHealthDataRecord.java

```diff
@@ -295,6 +297,19 @@ public void setUserDataGroups(Set<String> userDataGroups) {
         // DDB doesn't support empty sets, use null reference for empty set. This is also enforced by the builder.
         this.userDataGroups = (userDataGroups != null && !userDataGroups.isEmpty()) ? userDataGroups : null;
     }
+    
+    /** {@inheritDoc} */
+    @DynamoDBTypeConverted(converter=StringValueStringKeyMapMarshaller.class)
```

Comment:
Dynamo handles Maps just fine. Is there any particular reason we need to marshal to/from JSON?

Reply:
@DynamoDBTypeConvertedJson works perfectly fine, deleted this custom type converter in favor of that. Integration tests pass.